src/views/Registrations/DefineRegistration.vue

```diff
@@ -1,17 +1,100 @@
 <template>
   <div id="define-registration">
-    REGISTRATION PLACEHOLDER
+    <section class="mt-10">
+      <header id="name-info">
+        <h2>1. Name</h2>
```

Comment:
This might be a good time to try the CSS item number trick.

Reply:
I was meaning to ask about this. I lost the post. Do you happen to have a bookmark, 100% will try it out!

Reply:
https://chat.developer.gov.bc.ca/channel/registries-ui?msg=876LyFLQJk2KtnwG3

Reply:
Updated! Very easy solution, thank you 👍

Reply:
Looks good! Very simple. Might be worth demo'ing this at next tech demo?

Just curious: why isn't the first h2 "level 0" ?

Reply:
Its interesting that It doesn't increment unless we set the baseline so i'm assuming it counts the first instance of said element. 
ie the 1st `h2` is therefore `1`